toolset/setup/linux/languages/mono.sh

```diff
@@ -16,8 +16,7 @@ echo "deb http://jenkins.mono-project.com/repo/debian sid main" | sudo tee /etc/
 sudo apt-get update
 
 # Find the most recent snapshot
-#SNAPSHOT=$(apt-cache search 'mono-snapshot-.*-assemblies' | cut -d'-' -f3 | tail -1)
-SNAPSHOT="20150202010831"
+SNAPSHOT=$(apt-cache search 'mono-snapshot-.*-assemblies' | cut -d'-' -f3 | tail -1)
```

Comment:
Stated in another thread, but I would prefer to NOT using rolling snapshots like this; we often have to rerun tests weeks apart and there is no logging at any moment describing which version is being used.

I am fine updating our static snapshot number to a more recent version, but we cannot use this "use latest snapshot" implementation.